velox/connectors/hive/HiveConnectorSplit.h

```diff
@@ -49,6 +51,7 @@ struct HiveConnectorSplit : public connector::ConnectorSplit {
       dwio::common::FileFormat _fileFormat,
       uint64_t _start = 0,
       uint64_t _length = std::numeric_limits<uint64_t>::max(),
+      std::optional<FileProperties> _properties = std::nullopt,
```

Comment:
@acvictor Can you move the new parameter to end of the function?  This way it will not break presto build and easier to merge.

Reply:
Done.